src/pudl/transform/ferc1.py

```diff
@@ -6092,6 +6093,63 @@ class OtherRegulatoryLiabilitiesTableTransformer(Ferc1AbstractTableTransformer):
 }
 
 
+def filter_for_freshest_data_xbrl(df: pd.DataFrame, primary_keys) -> pd.DataFrame:
```

Comment:
If we want to use these for 714 also, we'll want to pull it into its own `pudl.transform.ferc_utils` module or something... but for now this is fine!

Reply:
yea... now that the 714 stuff got pulled into main i do think a new module is warrented.